hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/HoodieSparkMergeOnReadTable.java

```diff
@@ -105,6 +106,11 @@ public HoodieWriteMetadata<HoodieData<WriteStatus>> delete(HoodieEngineContext c
     return new SparkDeleteDeltaCommitActionExecutor<>((HoodieSparkEngineContext) context, config, this, instantTime, keys).execute();
   }
 
+  @Override
+  public HoodieWriteMetadata<HoodieData<WriteStatus>> deletePrepped(HoodieEngineContext context, String instantTime, HoodieData<HoodieRecord<T>> preppedRecords) {
```

Comment:
we don't need to explicitly override here if we use SparkDeletePreppedCommitActionExecutor for MOR as well

Reply:
Removed `deletePrepped` method here. Also removed `SparkDeletePreppedCommitActionExecutor`.

Reply:
Added back `deletePrepped` and `SparkDeletePreppedCommitActionExecutor` as `TestSqlStatement` started failing for MOR table.

Reply:
my bad. thanks